src/awesome_map/BoundaryInterceptor.js

```diff
@@ -225,6 +243,37 @@ define(function(require) {
             var eventType = event.type;
             var originalState;
 
+
+            // Boundary Detection
+            // @TODO: It seems that this event can be triggered by inertial events generated
```

Comment:
The original impetus for implementing these events is for Binders to use for switching tabs when scrolling beyond the top/bottom of the current tab.  In that case, for example, it might be kind of annoying if, while scrolling to the top of a large doc, residual inertial events upon reaching the top of the document caused the tab to change when the user desired only to reach the top of the document.

At the moment, I cant think of a clean way to handle the inertia without making the logic a lot more complicated, so i'll just make note of the potential weirdness to aid future debuggery.  I'm changing the TODO to just a Note, because i think it's still important to be aware of for anyone using these events.